deployments/deployment.yaml

```diff
@@ -31,6 +31,12 @@ objects:
         - env:
           - name: KEYCLOAK_SERVER
             value: "${KEYCLOAK_SCHEME}://${KEYCLOAK_HOST}:${KEYCLOAK_PORT}${KEYCLOAK_PATH}"
+          - name: SES_ACCESS_KEY
+            value: "${SES_ACCESS_KEY}"
+          - name: SES_SECRET_KEY
+            value: "${SES_SECRET_KEY}"
```

Comment:
Just curious if there's a reason we wouldn't want to drop these in secrets and reference them from there, now?

Reply:
100% fat finger on my part - updated to read from a secret marked as optional as to not break the current deployment.